packages/wallets/src/features/cashier/modules/FiatOnRamp/components/FiatOnRampProviderCard/FiatOnRampProviderCard.tsx

```diff
@@ -29,12 +29,7 @@ const FiatOnRampProviderCard: React.FC<TFiatOnRampProvider> = ({
                 <WalletText size='sm'>{description}</WalletText>
                 <div className='wallets-fiat-onramp-provider__icons'>
                     {paymentIcons.map((paymentIcon, index) => (
-                        <div
-                            data-testid={`dt_payment-method-icon-${paymentIcon.name}`}
-                            key={`payment-method-icon-${index}`}
-                        >
-                            {paymentIcon.icon}
-                        </div>
+                        <div key={`payment-method-icon-${index}`}>{paymentIcon.icon}</div>
```

Comment:
Pls avoid using index. React uses the key to determine which items have changed, are new, or are missing. If the keys are unstable, React might think that certain elements are new when they're not, leading to unnecessary re-renders. Looks like getPaymentIcons name are pretty unique. https://blog.anja-stricker.de/why-using-index-as-key-in-reactjs-map-function-can-cause-problems-and-how-to-fix-them. .

Reply:
Updated